setup.cfg

```diff
@@ -35,7 +35,19 @@ omit =
 
 [tool:pytest]
 filterwarnings = 
-	ignore: .*[The indexing module | get_statepoint | Use of.+as key] is deprecated.*: DeprecationWarning
+	error::FutureWarning:signac.*
```

Comment:
From the PR description, I understood that you wanted to identify all occurrences of `DeprecationWarning`. Could you briefly comment on the motivation for also treating occurrences of `FutureWarnings` as errors?

Is it just to have a clear overview of the progress of converting all warnings via the exceptions below?

Reply:
Marking all DeprecationWarnings as errors ensures that we can convert them to FutureWarnings. Marking all FutureWarnings as errors ensures that we don't leave anything behind in signac 2.0. This list should have no "ignores" when signac 2.0 is released (unless we decide to punt to 3.0 on some changes).

Reply:
Thanks for clarifying!